csrc/mlp.cpp

```diff
@@ -60,9 +90,11 @@ std::vector<at::Tensor> mlp_forward(int use_bias, int activation, std::vector<at
   auto reserved_size = get_mlp_reserved_space(batch_size, num_layers, output_features.data());
 
   // create output/workspace tensor
-  // TODO(deyuf): just get buffer?
+  // TODO(deyuf): just get buffer? why empty with .type() gets GPU tensor?
```

Comment:
is there an open TODO here?  If so, what?  If not, remove the TODO?

Reply:
removed now.